### DISCUSSION TITLE
Single HTML file of a book

### DISCUSSION THREAD

**Discussion by larmarange** [retypeapp/retype#84] [Ideas]
Do you have any plan for allowing to generate (in addition of the website) an HTML file (ideally self-contained) containing all the book allowing to propose to readers to download the overall book and/or to print it as a PDF with their browser.

**Comment by geoffreymcgill** [comment]
Converted this discussion to an official feature request so we can better track its progress. 

See https://github.com/retypeapp/retype/issues/85.
